### PR TITLE
[Technical Support] LPS-71856

### PR DESCRIPTION
**Pull request by dustinryerson**
/cc @jbalsas @blzaugg

https://issues.liferay.com/browse/LPS-71856

Hey @Robert-Frampton, this is a resend of https://github.com/Robert-Frampton/liferay-portal/pull/373 and https://github.com/blzaugg/liferay-portal/pull/178. I tried to keep the changes minimal while still fully fixing the issues present and maintaining the accessibility requirements detailed by Chema [here](https://github.com/blzaugg/liferay-portal/pull/178#issuecomment-294089512) and [here](https://github.com/blzaugg/liferay-portal/pull/178#issuecomment-294436909).

Please let me know if you have any questions, or if you'd rather I send this directly to Byran.

Thanks!

### CODE REVIEW

**Comment by dustinryerson** [comment]
@Robert-Frampton based on the results of https://github.com/Robert-Frampton/liferay-portal/pull/388#issuecomment-301550374, do you think I should split my fix into separate pulls and send them to the necessary repos / subrepos? What process do you recommend for this?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/text-html.ftl [hunk 1]
```diff
@@ -7,12 +7,22 @@
 	inputEditorName = "${namespacedFieldName}Editor"
 
 	fieldValue = paramUtil.getString(request, "${inputEditorName}", fieldValue)
+
+	toolbarSet = ""
```
Comment (Robert-Frampton) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/text-html.ftl]: Hey @dustinryerson, will it break if we just set the default value to "liferay" and remove the last `<#else>`  below?

**Comment by Robert-Frampton** [comment]
Hey @dustinryerson, yeah just send that commit to the forms and workflow team and the other two to me, then just comment on both that they are related.

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/text-html.ftl (hunk 1 above)
Comment (dustinryerson) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/text-html.ftl]: @Robert-Frampton that should work fine too - I tested that when I was making the fix and it works.

**Comment by dustinryerson** [comment]
Ok I'll send an updated pull with only the frontend changes, and I'll add a link to the forms and workflow pull.  I'll also make the modification you mentioned in your comment above.

Thanks @Robert-Frampton!

**Comment by Robert-Frampton** [comment]
Sounds good. Thanks, Dustin!

**Comment by Robert-Frampton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**